operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ServiceOperator.java

```diff
@@ -11,13 +11,19 @@
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.kubernetes.client.dsl.MixedOperation;
 import io.fabric8.kubernetes.client.dsl.ServiceResource;
+import io.strimzi.operator.common.Util;
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
 
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 /**
  * Operations for {@code Service}s.
  */
 public class ServiceOperator extends AbstractResourceOperator<KubernetesClient, Service, ServiceList, DoneableService, ServiceResource<Service, DoneableService>> {
+    private static final String CATTLE_ANNOTATION_SEED = "cattle.io";
```

Comment:
I think we should be more precise to have RANCHER in the const name because it's the only one having this annotation AFAIU, right?